TESTS/mbed_hal/lp_ticker/main.cpp

```diff
@@ -21,6 +21,10 @@
 #include "lp_ticker_api_tests.h"
 #include "hal/lp_ticker_api.h"
 
+#if DEVICE_DISABLED
```

Comment:
Hi
Is this only a temporary test ???
Adding a DEVICE_DISABLED feature seems not acceptable...

Reply:
Hi! No this test is not temporary. This PR provides a set of tests for the new HAL us/lp ticker API (requirements can be found in the ticker header files). These tests should pass on all mbed supported boards, unfortunately they fail on all tested boards ;-)
Now all the boards should be disabled and then all of the CIs run against that branch Israel and Oulu CI will fail because they expect certain boards to be present. These boards need to be re-enabled and I will port them to the new API.

I tried to disable the boards by removing release versions in `targets.json` or removing specified target from the list, but Travis says "no!" in both cases. So I provided this workaround.

Reply:
Says no means? It should work. @theotherjimmy suggestions?